src/dmd/backend/util2.d

```diff
@@ -109,12 +102,23 @@ static void __cdecl controlc_handler(void)
  * Trap control C interrupts.
  */
 
-#if !MARS
+version (MARS) { } else
+{
+
+version (CRuntime_DigitalMars)
```

Comment:
This duplicates version CRuntime_DigitalMars above.

Reply:
ok